Tiltfile

```diff
@@ -33,6 +33,10 @@ Here's a quick rundown of these services and their properties:
 
 enable_feature("events")
 
+load('ext://restart_process', 'docker_build_with_restart')
+update_settings(5)
```

Comment:
how would you feel about passing 5 as a named arg? it reads pretty weirdly as-is

Reply:
eh i just removed the call because i didn't feel like figuring out where to put it 😅